blueprints/apigee/apigee-x-foundations/README.md

```diff
@@ -0,0 +1,469 @@
+# Apigee Foundations
+
+This module creates all the resources necessary to set up Apigee X on Google Cloud. Find below a few examples of different Apigee architectures that can be created using this module.
+
+Apigee can be exposed to clients using Regional Internal Application Load Balancer, Global External Application Load Balancer or both. When using the Regional Internal Application Load Balancer, self-managed certificates (incuding self-signed certificates generated in this same module) can be used. When using the Global External Application Load Balancer Google-managed certificates or self-managed certificates (including self-signed certificates generated in this same module).
+
```

Comment:
Can you add a TOC here? And then I would make section headings below shorter, maybe by abbreviating the LB names?

Reply:
done